docs/book/src/topics/ignition-support.md

```diff
@@ -51,22 +91,7 @@ spec:
 See [Using clusterawsadm to fulfill prerequisites](./using-clusterawsadm-to-fulfill-prerequisites.md) for more
 details.
 
-## Enabling EXP_BOOTSTRAP_FORMAT_IGNITION feature gate
```

Comment:
I might have missed, but is the feature gate removed for ignition support?

Reply:
I haven't removed it 👍 it is just a bit higher up in the doc now